tests/staticfiles_tests/test_storage.py

```diff
@@ -360,6 +360,14 @@ def test_post_processing_nonutf8(self):
         self.assertEqual("Post-processing 'nonutf8.css' failed!\n\n", err.getvalue())
         self.assertPostCondition()
 
+    def test_css_data_uri_with_nested_url(self):
```

Comment:
I wonder if this test would be better at line 238. That would put it with the other CSS tests before the group of CSS source map tests.